service/src/main/resources/config/test/plugin/test.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name" : "test",
```

Comment:
Glad to see you got this working! It looks great. Can you please move it under `src/test/resources` instead of `src/main/resources` though?

Reply:
Will `application-test.yml` load underlays from `src/test`?

Reply:
No it doesn't, but I think you can follow the example of loading test underlays from a different place in the `indexer` tests (e.g. [here](https://github.com/DataBiosphere/tanagra/blob/main/indexer/src/test/java/bio/terra/tanagra/indexing/deserialization/InvalidUnderlayTest.java)).

Reply:
So I need to create a new application profile under test, set that as active, create an underlay config directory in test, and then override the parent directory in the test class?

Reply:
The example test class that I linked above does not contain a new application profile -- can you clarify why you'd like to do that?

Given that you're testing underlay config file validation, I was suggesting that you just add a new class in that same package and mimic how it loads underlays from a `src/test/resources` directory.

Reply:
I thought I needed to add the underlay to the `tanagra.underlay.files` parameter in the application profile in order for it to be loaded?

Reply:
For running the application, yes that's how you specify the underlays. But when testing the underlay config validation, you don't need to go through the Spring framework for this I don't think. The example test classes above don't rely on that property for the invalid underlays they load. e.g. `underlay/PrimaryEntityNotFound.json` is not in any application yaml file.

Reply:
Just put up a PR against this branch. PTAL